Gemfile.lock

```diff
@@ -271,17 +271,17 @@ GEM
       railties (>= 5.0.0)
     faker (2.15.1)
       i18n (>= 1.6, < 2)
-    faraday (1.1.0)
+    faraday (1.2.0)
       multipart-post (>= 1.2, < 3)
       ruby2_keywords
     faraday_middleware (1.0.0)
       faraday (~> 1.0)
-    fastimage (2.1.1)
+    fastimage (2.2.0)
     fastly (2.5.3)
     feedjira (3.1.2)
       loofah (>= 2.3.1)
       sax-machine (>= 1.0)
-    ffi (1.13.1)
+    ffi (1.14.2)
```

Comment:
depedency of a bunch fo stuff, nothing to report, https://github.com/ffi/ffi/blob/master/CHANGELOG.md indicates new methods and bug fixes